packages/web-pkg/src/components/sideBar/Spaces/SpaceInfo.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="space_info">
+  <div class="space_info oc-px-s oc-py-m">
```

Comment:
```suggestion
  <div class="space_info oc-p-s">
```

Same as for `FileInfo`

Reply:
see also https://github.com/owncloud/web/pull/8398#discussion_r1099996395